docs/source/quick-start.ipynb

```diff
@@ -13,7 +13,16 @@
    "cell_type": "code",
```

Comment:
<p>probably not needed in the quickstart notebook. Would a print statement above do instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xarray-contrib/xskillscore/pull/370/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>

Reply:
<p>I want to show they are not identical. but they are very close here.</p>